components/brave_ads/browser/ads_service_impl.h

```diff
@@ -423,6 +426,8 @@ class AdsServiceImpl : public AdsService,
 
   SimpleURLLoaderList url_loaders_;
 
+  NotificationMetrics notification_metrics_;
```

Comment:
NotificationAdMetrics notification_ad_metrics_;

Reply:
removed class completely, since we no longer need debounce logic